### DISCUSSION TITLE
Smart Meter: time controlled request

### DISCUSSION THREAD

**Discussion by Yunkie** [arendst/Tasmota#19506] [Q&A]
Hi there,

i want to read out the EMH LZQJ-XC. I do it with Tasmota 13.1.0 on an ESP32 (Olimex PoE) and it work fine for the total counter.
But to get the active power, I need to send another initialization sequence and I don't know how to switch in the script. I thought maybe it would be possible to send either one or the other sequence depending on the time. 

Then I would get the total counter reading only once an hour and the active power the rest of the time. Is that possible?

Here is the Script:
```
>D
>B
=>sensor53 r

>M 1
+1,3,o,0,300,,1,100,2F3F210D0A
;+1,3,o,0,300,,1,100,2F34210D0A <-- Thats the part for the active power reading

1,0.0.9(@#),Zähler-Nr.,,Zähler-Nr.,0
1,1-1:1.8.1(@1,Bezug_HT,kWh,Bezug_HT,3
1,1-1:1.8.2(@1,Bezug_NT,kWh,Bezug_NT,3
1,1-1:2.8.1(@1,Einsp._HT,kWh,Einsp._HT,3
1,1-1:2.8.2(@1,Einsp._NT,kWh,Einsp._NT,3
; Here I still have to enter the correct OBIS codes for the active power
#
```

Thanks for help!

**Comment by Weltkugel** [comment]
Hi,
in the SML-wiki is an example to change the baudrate using a specific character string.

> For sending a specific character string:
> sml(METERNUMBER 1 STRING)

You could send this string in >S to get the active power.

**Comment by gemu2015** [comment]
you may simply add the alternate request to the meter definition
now every 10 seconds the response will change. 
+1,3,o,0,300,0,1,100,2F3F210D0A,2F34210D0A

sending the alternate request only once an hour is also possible but more effort, see example in the docs
